packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -214,36 +214,90 @@
 <section data-test-percy>
   <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
 
-  <p class="dummy-paragraph"><strong>Basic usage:</strong></p>
+  <h4 class="dummy-h4">Tab</h4>
+
+  <h5 class="dummy-h6">Content</h5>
+  <div class="dummy-tab-base-sample">
+    <div>
+      <span class="dummy-text-small">Text only</span>
+      <br />
+      <Hds::Tabs::Tab>Lorem ipsum</Hds::Tabs::Tab>
+    </div>
+    <div>
+      <span class="dummy-text-small">Icon + Text</span>
+      <br />
+      <Hds::Tabs::Tab @icon="hexagon">Lorem ipsum</Hds::Tabs::Tab>
+    </div>
+    <div>
+      <span class="dummy-text-small">Text + Counter</span>
+      <br />
+      <Hds::Tabs::Tab @count="10">Lorem ipsum</Hds::Tabs::Tab>
+    </div>
+    <div>
+      <span class="dummy-text-small">Icon + Text + Counter</span>
+      <br />
+      <Hds::Tabs::Tab @icon="hexagon" @count="10">Lorem ipsum</Hds::Tabs::Tab>
+    </div>
+  </div>
+
+  <h5 class="dummy-h6">States</h5>
+  <div class="dummy-tab-base-sample">
+    {{#each @model.TAG_STATES as |state|}}
+      <div class="dummy-tab-base-sample-item">
+        <span class="dummy-text-small">{{capitalize state}}:</span>
+        <Hds::Tabs::Tab mock-state-value={{state}} mock-state-selector="a">Lorem ipsum</Hds::Tabs::Tab>
+        <Hds::Tabs::Tab @icon="hexagon" @count="10" mock-state-value={{state}} mock-state-selector="a">Lorem ipsum</Hds::Tabs::Tab>
+      </div>
+    {{/each}}
```

Comment:
Here instead I've showcased the different kinds of "interactive states" for the component, using the "mock" technique mentioned above. See how it's done in a similar way in (almost) all the other components (look for `Status` or `States` in their showcase).

(I've created two rows, one with just text and one with icon + text + counter, so we can test both, and it's easier to detect possible differences/inconsistencies in the layout)